opdracht-1.js

```diff
@@ -0,0 +1,44 @@
+console.log("..............1A..............");
+// 1a
+const tvType = inventory.map((tv) => {
+    tv = tv.type
+    return tv
+});
+
+console.log(tvType);
+
+console.log("..............1B..............");
+// 1b
+const soldOut = inventory.filter((stock) => {
+    if(stock.originalStock - stock.sold === 0) {
+        return stock
+    }
+});
+
+console.log(soldOut);
+
+console.log("..............1C..............");
+// 1c
+const ambiLight = inventory.filter((hasAmbiLight) => {
+    if(hasAmbiLight.options.ambiLight === true) {
+        return hasAmbiLight
+    }
+});
```

Comment:
`ambiLight` is van zichzelf al een boolean en vereist dus geen expliciete check:
```suggestion
const tvsWithAmbiLight = inventory.filter(tv => tv.options.ambiLight);
```